.github/workflows/bidi-scanner.yml

```diff
@@ -0,0 +1,14 @@
+name: Run Unicode Control Character Scan
```

Comment:
Because this step shows up in `repository-scanner.yml` below as well, does this mean that the step will run twice?